documentation/how-to/measuring-tool.en.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Measuring tool
+tx_slug: documentation_how-to_measuring-tool
+---
+
+# Measuring tool
+
+QField offers measurement functionalities out of the box. To enable the measuring tool, open the main menu and select *Measuring Tool*.
+
+!![](../assets/images/measuring_tool.png)
+
+Once the tool is enabled, simply use the digitizing controls located at the lower-right corner of the screen to add and remove
+vertices. By default, the measured geometry will be a line; to change to a polygon, simply connect the coordinate cursor to the first
+vertex entered.
+
+For the segment formed of the two last vetices added, details returned include segment length and its azimuth. When the measured
+geometry is a line, the total line length is provided while the perimeter and area are displayed for polygons.
+
+## Elevation profiling
+
+When the measuring tool is enabled, an *Elevation Profile* tool button appears at the top-right corner of the screen. Toggling
+this on will open the elevation profiling panel, which will show the terrain elevation as well as interesecting vector
+features along the measured geometry.
+
+!![](../assets/images/elevation_profiling.png)
+
+For ease of use, QField adds a default 30-meter resolution terrain. It is however possible to customize terrain settings through
```

Comment:
Link this to a description of the data source?

Reply:
@m-kuhn , I've looked around before opening the PR but couldn't find a good landing page we could forward people to here.

Reply:
@m-kuhn , alright, here's a compromise: I've added an explicit reference to the dataset we're using here (i.e. the Mapzen Global Terrain). Hope that's enough.